README.md

```diff
@@ -117,36 +127,37 @@ first, so that the configuration script will find this.
 Also install x265 and its development files if you want to use HEIF encoding.
 
 The basic build steps are as follows:
-````
+
+````sh
 mkdir build
 cd build
 cmake --preset=release ..
 make
```

Comment:
This still implies a dependency on 'make' on some sytems that e.g. would use Makefiles generator by default, but perhaps Ninja instead...

Better use agnostic `cmake --build .`

Reply:
The subject PR is orthogonal to that topic. I have no objections to that kind of change, but never use `cmake --build .` myself.